core/src/models/App.ts

```diff
@@ -238,7 +238,7 @@ export class App extends LoggedModel<App> {
   async getConfigObject() {
     const { type, name } = this;
 
-    const options = await this.getOptions();
+    const options = await this.getOptions(false);
```

Comment:
Nice! That `sourceFromEnvironment` argument /was/ helpful after all!

Reply:
[Thanks, Past Evan!](https://github.com/grouparoo/grouparoo/pull/1533)